db/migrate/20181017170747_create_projects.rb

```diff
@@ -2,7 +2,7 @@ class CreateProjects < ActiveRecord::Migration[5.2]
   def change
     create_table :projects do |t|
       t.string :name
-      t.integer :user_id
+			t.integer :workspace_id
```

Comment:
NEVER edit existing migrations
please create new migration which removes user_id column and adds workspace_id